VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -1,92 +1,80 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@color/edx_grayscale_neutral_white">
+    android:background="@color/edx_grayscale_neutral_white"
+    android:orientation="vertical">
 
-    <RelativeLayout
+    <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        android:paddingTop="@dimen/discussion_responses_box_padding"
+        android:orientation="vertical"
+        android:paddingBottom="3dp"
         android:paddingLeft="@dimen/discussion_responses_box_padding"
+        android:paddingStart="@dimen/discussion_responses_box_padding"
         android:paddingRight="@dimen/discussion_responses_box_padding"
-        android:paddingBottom="3dp">
+        android:paddingEnd="@dimen/discussion_responses_box_padding"
+        android:paddingTop="@dimen/discussion_responses_box_padding">
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/discussion_responses_thread_row_title_text_view"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_alignParentTop="true"
-            android:textSize="@dimen/edx_base"
-            android:textColor="@color/edx_grayscale_neutral_x_dark"
             android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
+            android:textColor="@color/edx_grayscale_neutral_x_dark"
+            android:textSize="@dimen/edx_base"
             tools:text="Thread title" />
 
         <org.edx.mobile.view.custom.ETextView
             android:id="@+id/discussion_responses_thread_row_body_text_view"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
             android:minHeight="50dp"
-            android:layout_below="@+id/discussion_responses_thread_row_title_text_view"
-            android:textSize="@dimen/edx_small"
             android:textColor="@color/edx_grayscale_neutral_dark"
-            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
-            tools:text="Thread body"/>
+            android:textSize="@dimen/edx_small"
+            tools:text="Thread body" />
 
-        <RelativeLayout
+        <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/discussion_responses_thread_row_body_text_view">
+            android:orientation="horizontal">
```

Comment:
Could you add maybe a `android:gravity="center_vertical"` here or set `layout_gravity` on the icons? It looks like they are not currently aligned vertically.
![screen shot 2015-10-16 at 1 02 50 pm](https://cloud.githubusercontent.com/assets/1685613/10548087/77fc8e28-7406-11e5-9f81-0a699227984a.png)